docroot/modules/custom/va_gov_backend/src/Plugin/Block/SidebarMenusBlock.php

```diff
@@ -24,6 +25,19 @@
  * )
  */
 class SidebarMenusBlock extends BlockBase implements ContainerFactoryPluginInterface {
+  const LOVELL_SECTIONS = [
+    '1040' => 'va',
+    '1039' => 'tricare',
+    '347' => 'both',
+  ];
```

Comment:
```suggestion
  const TRICARE_ID = '1039';
  const TRICARE_NAME = 'tricare';
  const VA_ID = '1040';
  const VA_NAME = 'va';
  const BOTH_ID = '347';
  const BOTH_NAME = 'both';
  const LOVELL_SECTIONS = [
    self::VA_ID => self::VA_NAME,
    self::TRICARE_ID => self::TRICARE_NAME,
    self::BOTH_ID => self::BOTH_NAME,
  ];
```
Go all in with const